packages/uhk-web/src/app/components/svg/module/svg-module.component.html

```diff
@@ -16,5 +16,6 @@
             (capture)="onCapture(i, $event)"
             (mouseenter)="onKeyHover(i, $event, true)"
             (mouseleave)="onKeyHover(i, $event, false)"
+            [attr.tabindex]="-1"
```

Comment:
line 10 also contains the tab index settings. pls modify that.

And I think not need the one way data binding because the `-1` is constant value

```suggestion
            tabindex="-1"
```